Traceback (most recent call last):
  File "/home/lx/Udesc/BAN2/python/as/interface/masterMind.py", line 41, in insereModelos
    insert(c,tabelas,what)
  File "/home/lx/Udesc/BAN2/python/as/interface/banco.py", line 196, in insert
    c.execute("insert into "+tabelas+" values "+what+");")
  File "/usr/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
_mysql_exceptions.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'i' at line 1")
(123,3123,12312)
modelos
(,,)
modelos
(4213,423423,42342)
modelos
Traceback (most recent call last):
  File "/home/lx/Udesc/BAN2/python/as/interface/masterMind.py", line 40, in insereModelos
    insert(c,tabelas,what)
NameError: global name 'insert' is not defined
Traceback (most recent call last):
  File "/home/lx/Udesc/BAN2/python/as/interface/masterMind.py", line 40, in insereModelos
    self.insert(c,tabelas,what)
  File "/home/lx/Udesc/BAN2/python/as/interface/banco.py", line 197, in insert
    c.execute("insert into "+tabelas+" values "+what+";")
  File "/usr/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
_mysql_exceptions.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'i' at line 1")
Traceback (most recent call last):
  File "/home/lx/Udesc/BAN2/python/as/interface/masterMind.py", line 67, in insertFuncionario
    self.insert(c,tabelas,what)
  File "/home/lx/Udesc/BAN2/python/as/interface/banco.py", line 200, in insert
    c.execute("insert into %s values %s;"%(tabelas,what))
  File "/usr/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
_mysql_exceptions.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''asdsa')' at line 1")
Traceback (most recent call last):
  File "/home/lx/Udesc/BAN2/python/as/interface/masterMind.py", line 67, in insertFuncionario
    self.insert(c,tabelas,what)
  File "/home/lx/Udesc/BAN2/python/as/interface/banco.py", line 200, in insert
    c.execute("insert into %s values %s;"%(tabelas,what))
  File "/usr/lib/python2.7/site-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib/python2.7/site-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
_mysql_exceptions.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''asdsa')' at line 1")
